conctr.agent.lib.nut

```diff
@@ -195,18 +228,393 @@ class Conctr {
     }
 
 
-    //
+    // 
+    // Sets the protocal that should be used 
+    // 
+    // @param   {String}    protocol Either amqp or mqtt
+    // @return  {String}    current protocal after change
+    // 
+    function setProtocol(protocol) {
+        if (protocol == MQTT) {
+            _protocol = protocol;
+        } else {
+            server.error(protocol + " is not a valid protocol.");
```

Comment:
We should probably have an `error` version of `log`.
Maybe `function log(msg, iserror = false)`